src/webgpu/gpu_test.ts

```diff
@@ -887,14 +913,24 @@ export class GPUTestBase extends Fixture<GPUTestSubcaseBatchState> {
             buffer: storageBuffer,
           },
         },
+        {
+          binding: 2,
+          resource: {
+            buffer: uniformBuffer,
+          },
+        },
       ],
     });
 
     const encoder = this.device.createCommandEncoder();
     const pass = encoder.beginComputePass();
     pass.setPipeline(computePipeline);
     pass.setBindGroup(0, uniformBindGroup);
-    pass.dispatchWorkgroups(1);
+    pass.dispatchWorkgroups(
+      (width + kWorkgroupSizeX - 1) / kWorkgroupSizeX,
+      (height + kWorkgroupSizeY - 1) / kWorkgroupSizeY,
```

Comment:
Use Math.floor here to explicitly get an integer (instead of relying on WebIDL conversion from double to integer)